support-frontend/assets/pages/paper-subscription-checkout/components/paperCheckoutForm.test.tsx

```diff
@@ -234,8 +234,8 @@ describe('Newspaper checkout form', () => {
 			expect(
 				screen.queryAllByText(
 					'Please use only letters, numbers and punctuation.',
-				),
-			).toBeTruthy();
+				).length,
+			).toBeGreaterThan(0);
```

Comment:
This test wasn’t actually working correctly, I’ve updated it to match the others (and verified that it fails when expected to)